.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,13 @@
+name: Auto-approve Renovate's PRs
+
+on:
+  pull_request_target
+
+jobs:
+  auto-approve-renovate-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hmarr/auto-approve-action@v3.2.1
```

Comment:
To avoid security risk, shouldn't we fork auto-approve-action and use our forked version?

Reply:
I made another commit that ignores using third-party action and uses GitHub's `gh` command instead.